projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/vdk-jupyterlab-extension/vdk_ui.py

```diff
@@ -93,5 +93,5 @@ def create_job(
         if local:
             cmd.validate_job_path(path, name)
 
-        cmd.create_job(name, team, path, cloud, local)
+        cmd.create_job(name, team, path, bool(cloud), bool(local))
```

Comment:
I like the good contract, using the data types accordingly. The way `create_job()` was accepting boolean in favour of string (anything could be represented as string).

My suggestion is, to parse boolean from string before invoking `create_job()`, for example:
```bool(input_data[VdkOption.CLOUD.value])```

Reply:
fixed